app/services/fast_track/hypertension_special_issue_manager.rb

```diff
@@ -8,27 +8,31 @@ def initialize(submission)
       @submission = submission
     end
 
-    def add_special_issue
-      data = JSON.parse(submission.form_json, symbolize_names: true)
-      disabilities = data[:form526][:form526][:disabilities]
-      data[:form526][:form526][:disabilities] = add_rrd_to_disabilities(disabilities)
-      submission.update!(form_json: JSON.dump(data))
+    def submission_data
+      @submission_data ||= JSON.parse(submission.form_json, symbolize_names: true)
```

Comment:
Is it an option to just move the JSON parse logic into the initializer and get rid of this method completely?

```
    def initialize(submission)
      @submission = submission
      @submission_data ||= JSON.parse(submission.form_json, symbolize_names: true)
    end